packages/pds/tests/views/__snapshots__/author-feed.test.ts.snap

```diff
@@ -71,8 +71,11 @@ Array [
               Object {
                 "alt": "tests/image/fixtures/key-landscape-small.jpg",
                 "image": Object {
-                  "cid": "cids(3)",
+                  "$type": "blob",
                   "mimeType": "image/jpeg",
+                  "ref": Object {
+                    "/": "cids(3)",
+                  },
```

Comment:
Since in the record this is an `'image'` lex type, should we have `"$type": "image"` here?  Or is the story there that those lex types (also e.g. `'video'`) are just for validation on write?

Reply:
Yeah I put a note about that in the PR description. My original intention was to do `$type: image`, but I wasn't seeing any benefit out of it. You get everything you need from the mimetype. And it was making certain things - like using the output of upload.blob as an input for an image type - kinda annoying & complex. I see the lex type mainly being useful for validation.

Reply:
I'm down with that 👍